CRM/Contact/Form/Task/PDFTrait.php

```diff
@@ -151,7 +151,7 @@ public function addPDFElementsToForm(): void {
     $form->assign('totalSelectedContacts', !is_null($form->_contactIds) ? count($form->_contactIds) : 0);
 
     $form->add('select', 'document_type', ts('Document Type'), CRM_Core_SelectValues::documentFormat());
-
+    $this->assign('uploadFormatsSupported', CRM_Utils_PDF_Document::$ooxmlMap);
```

Comment:
This isn't right. It's an array.

Reply:
oh - I need this sillyness = `implode(', .', array_keys(CRM_Utils_PDF_Document::$ooxmlMap))`

Reply:
OK - pushed it up

Reply:
Thanks, and yes it's silly, but this still breaks it:

<img width="423" alt="screenshot" src="https://github.com/civicrm/civicrm-core/assets/2967821/00455f9b-fdcf-4974-a450-4e9f4a54adf1">

Reply:
@demeritcowboy I don't suppose I can just drop it?

Reply:
@demeritcowboy I just put the text in - this isn't configurable so why make it so complicated?